backend/reg.mli

```diff
@@ -63,17 +64,21 @@ and stack_location =
 val equal_location : location -> location -> bool
 
 val dummy: t
+
 val create: Cmm.machtype_component -> t
+val create_with_typ: t -> t
+val create_with_typ_and_name: ?prefix_if_var:string -> t -> t
+val create_at_location: Cmm.machtype_component -> location -> t
+
 val createv: Cmm.machtype -> t array
-val createv_like: t array -> t array
-val clone: t -> t
-val at_location: Cmm.machtype_component -> location -> t
+val createv_with_id: id:Ident.t -> Cmm.machtype -> t array
+val createv_with_typs: t array -> t array
+val createv_with_typs_and_id: id:Ident.t -> t array -> t array
```

Comment:
(I` wonder whether the `createv`
functions could use a longer
name, to avoid the `v`.)